packages/api/src/router/notification.ts

```diff
@@ -0,0 +1,54 @@
+import type { ExpoPushMessage } from "expo-server-sdk";
+import { Expo } from "expo-server-sdk";
+import { z } from "zod";
+
+import { createTRPCRouter, protectedProcedure } from "../trpc";
+
+const expo = new Expo({ accessToken: process.env.EXPO_ACCESS_TOKEN });
+
+export const notificationRouter = createTRPCRouter({
+  push: protectedProcedure
+    .input(
+      z.object({
+        pushTokens: z.array(z.string()),
+        title: z.string().optional(),
+        body: z.string().optional(),
+        subtitle: z.string().optional(),
+        // TODO: add optional data payload
+      }),
+    )
```

Comment:
this might be nice to keep for testing notifications in the future 🤔 -- I'm assuming that the blocker before integrating this with something like the newPost endpoint is that we don't have the user pushTokens stored in the db right now?